### ISSUE TITLE
1.1.0 navbar alignment

### ISSUE THREAD

**Issue by j4mie** [bigskysoftware/missing#32]
Hi, thanks for missing.style!

I seem to have a regression between 1.0.9 and 1.1.0 with the following markup:

```html
    <header class="navbar">
      <div>
        <a href="/" class="allcaps">
          Reactor⚡
        </a>
      </div>
      <nav aria-label="Dashboard sections">
        <ul role="list">
          <li>
            <a href="/climate/">
              Climate
            </a>
          </li>
          <li>
            <a href="/energy/">
              Energy
            </a>
          </li>
          <li>
            <a href="/system/logs/">
              System
            </a>
          </li>
        </ul>
      </nav>
    </header>
```

On 1.0.9 it looks like this:

<img width="846" alt="image" src="https://github.com/bigskysoftware/missing/assets/6988/64c2f006-0e16-486f-8cfb-3f19d59bdb93">

On 1.1.0 it looks like this:

<img width="848" alt="image" src="https://github.com/bigskysoftware/missing/assets/6988/00a72989-3727-4cb8-91e1-b6b167bbb1aa">

(in other words, the navbar is now right-aligned rather than centred).

**Comment by dz4k** [comment]
I can't reproduce on the latest version, can you check if it's been fixed?

**Comment by j4mie** [comment]
@dz4k still the same for me on 1.1.1. Is there anything I can do to help debug?

**Comment by dz4k** [comment]
@j4mie could you make a minimal reproduction on CodePen or somewhere similar?

**Comment by graf0** [comment]
have the same problem with 1.1.1 - but only of Firefox, on Edge it's centered.

**Comment by graf0** [comment]
it seems that under edge following style is applyied to last child of nav:
![obraz](https://github.com/bigskysoftware/missing/assets/645149/90ad6e5c-cb1d-4929-9dff-7a648101f65f)

But on firefox - it's not applied.

**Comment by j4mie** [comment]
Hopefully I've done this right: https://codepen.io/j4miem/pen/ExMePqK

Agreed with @graf0 that it seems to be Firefox-specific issue. Works fine in Chrome and Safari.

**Comment by Fingel** [comment]
Does anyone know a workaround for this? I thought it was default behavior since I use Firefox, spent a while trying to fix it.

**Comment by graf0** [comment]
> Does anyone know a workaround for this? I thought it was default behavior since I use Firefox, spent a while trying to fix it. 
> 
> The issue appears after upgrading from 1.0.13 to 1.1.0.

Try to add margin-right:auto to last element in navbar

**Comment by dz4k** [comment]
this was an artifact of our compiler being too overzealous with backcompat and replacing modern css with stuff that doesn't work. should be fixed in 1.1.2 (releasing soon)

**Comment by j4mie** [comment]
❤️ thank you!
